using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -872,6 +1027,49 @@ try {
 }
 ```
 {% endtab %}
+
+{% tab title="Cordova" %}
+
+### getUrlVariables
+
+#### Cordova
+
+{% hint style="info" %}
+This method was added in react-native-acpcore v1.0.5.
```

Comment:
is this reference to react-native correct?

Reply:
whoops, thanks for catching this. i'll fix it as well.